models/userDao.js

```diff
@@ -0,0 +1,24 @@
+const { appDataSource } = require("./dbconfig");
+
+const createUser = async (email, password, name, phone_number) => {
+  try {
+    return await appDataSource.query(
```

Comment:
```suggestion
    return appDataSource.query(
```
동기적으로 실행될 필요가 없기 때문에 await가 필요없습니다.